lib/ansible/modules/commands/xmlrpc_client.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Stefan Midjich <swehack at gmail dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'curated'
+}
+
+DOCUMENTATION = '''
+---
+module: xmlrpc_client
+
+short_description: Run XMLRPC commands
+
+version_added: "2.4"
+
+description:
+    - This module lets you establish an XMLRPC session and call any method
+      by its xmlrpc path. On dry run this module will only establish API
+      connection and check if your method exists in the method list but not
+      call it.
+
+options:
+    url:
+        description:
+            - The API URL you connect to.
+        required: true
+    path:
+        description:
+            - The RPC path/method you are calling.
+        required: true
+    args:
+        description:
+            - A list of arguments for the RPC call. Placed first in any RPC call.
+        required: false
+    kwargs:
+        description:
+            - A dictionary of arguments for the RPC call. Comes after the args list.
+        required: false
```

Comment:
This is not needed for parameters that are not required. Guideline dictates to leave it out.